clue.js

```diff
@@ -0,0 +1,316 @@
+// OBJECTS FOR ALL THE SUSPECTS
+
+const mrGreen = {
+  firstName: 'Jacob',
+  lastName: 'Green',
+  color: "#448c4a",
+  description: 'He has a lot of connections',
+  age: 45,
+  image: './assets/green.png',
+  occupation: 'Entrepreneur',
+}
+
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: '#8b539a',
+  description: 'He talks much but say little',
+  age: 45,
+  image: './assets/plum.png',
+  occupation: 'Visting STEM professor',
+}
+
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: '#ff8d48',
+  description: 'She can stare down a lion',
+  age: 30,
+  image: './assets/scarlet.png',
+  occupation: 'Manager of the Batman Zoo',
+}
+
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: '#7775c0',
+  description: 'She is stronger than you',
+  age: 37,
+  image: './assets/peacock.png',
+  occupation: 'Yoga teacher',
+}
+
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: '#e6b127',
+  description: "He was part of Franco's gang",
+  age: 73,
+  image: './assets/mustard.png',
+  occupation: 'Colonel',
+}
+
+const mrsWhite = {
+  firstName: 'Rose',
+  lastName: 'White',
+  color: '#e1d2a9',
+  description: 'She will not miss one thing',
+  age: 63,
+  image: './assets/white.png',
+  occupation: 'Facility consultant',
+}
+
+// OBJECTS FOR ALL THE WEAPONS
+
+const rope = {
+  name: 'Rope',
+  weight: 10,
+  id: "Rope",
+  image: './assets/rope.png'
+}
+
+const knife = {
+  name: 'Knife',
+  weight: 2,
+  id: "Knife",
+  image: './assets/knife.png'
+}
+
+const candlestick = {
+  name: 'Candlestick',
+  weight: 10,
+  id: "Candlestick",
+  image: './assets/candlestick.png'
+}
+
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 16,
+  id: "Dumbbell",
+  image: './assets/dumbell.png'
+}
+
+const poison = {
+  name: 'Bottole of Poison',
+  weight: 0.1,
+  id: "Poison",
+  image: './assets/poison.png'
+}
+
+const axe = {
+  name: 'Axe',
+  weight: 5,
+  id: "Axe",
+  image: './assets/axe.png'
+}
+
+const bat = {
+  name: 'Bat',
+  weight: 8,
+  id: "Bat",
+  image: './assets/bat.png'
+}
+
+const trophy = {
+  name: 'Trophy',
+  weight: 8,
+  id: "Trophy",
+  image: './assets/trophy.png'
+}
+
+const pistol = {
+  name: 'Pistol',
+  weight: 3,
+  id: "Pistol",
+  image: './assets/pistol.png'
+}
+
+// OBJECTS FOR ALL THE ROOMS
+
+const diningRoom = {
```

Comment:
Great idea to turn the rooms into objects, so you could add images for them aswell :+1: